tests/unittests/tests-checksum/tests-checksum-fletcher16.c

```diff
@@ -14,6 +14,10 @@
 
 #include "tests-checksum.h"
 
+/* String is longer than 20 so it checks the wrap-around property of
+ * fletcher16 */
+static const unsigned char wrap_around_data[] = "u9k89rm88z58f3hGxTfgG3BnyKnyRP";
```

Comment:
Would be probably best then to change this from `unsigned char` to `uint8_t`, even in the API ;-)